src/containers/ArticlePage/LearningResourcePage/components/LearningResourceForm.tsx

```diff
@@ -36,6 +37,7 @@ import { FlexWrapper, MainContent } from "../../styles";
 
 interface Props {
   article?: IArticle;
+  articleHistory?: UseQueryResult<IArticle[]>;
```

Comment:
Er det dumt å sende `UseQueryResult` rundt på denne måten?
Trenger å refetche på lagring pga notater, men følte det også var litt teit å legge fetch-hooken i `useArticleFormHooks` når vi allerede har en hook for fetching av data.